include/seqan3/core/debug_stream/debug_stream_type.hpp

```diff
@@ -12,7 +12,7 @@
 
 #pragma once
 
-#include <iosfwd>
+#include <ostream>
```

Comment:
This now counter-acts the purpose of a lightweight debug_stream_type?

Reply:
Kind of
I can do different includes for libstdc++ (gcc) and libc++ (llvm) :D

Reply:
there is no iosfw equivalent for libc++?

I would vote for different includes then :)

Reply:
I'll have to check what the issue was, but we are accessing stuff that requires a defined `std::ostream`. Whatever we need is apparently in `<iosfwd>` for libstdc++, but not in libc++

Reply:
Actually, we don't need `<ostream>` here, but in `pretty_printing.hpp`. 

So, good catch :+1: